library/kotlin/io/envoyproxy/envoymobile/EngineBuilder.kt

```diff
@@ -239,7 +257,7 @@ open class EngineBuilder(
         EngineImpl(
           engineType(),
           EnvoyConfiguration(
-            statsDomain, connectTimeoutSeconds,
+            statsDomain, statsdPort, connectTimeoutSeconds,
```

Comment:
Can we rename public references to statsDomain to be grpcStatsDomain, since we're distinguishing between sinks/protocols now?

Also can we have some programmatic guard/indication that you're doing it wrong if both are set?

Reply:
Done and done